src/middleware/navigation/index.ts

```diff
@@ -17,5 +18,6 @@ export const navigation = {
   hasBeneficialOwnersStatement,
   hasBOsOrMOs,
   hasOverseasName,
-  hasTrust
+  hasTrust,
+  hasOverseasEntityNumber
```

Comment:
sorry for a late comment! 
Should we put this in our own `update.navigation.ts` file then in the `routes/index.ts` import your new `hasOverseasEntityNumber` from `update.navigation.ts`? Keep the navigation entities separate between the 2 journeys?

I was just looking at this in my own ticket.

Reply:
or put it under another `const` named `updateNavigation` in this file?

Reply:
@corybudd-ch , or we could add the navigation entries for update in an update folder , creating new navigation middleware seems to be a pattern we have not followed for other middleware, @AndreKramer7654 any suggestions?

Reply:
have separated out the update middleware checks into update folder .